xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
-      </row>
-      <row>
-       <entry>
-        <para>
-         File Sel.
-        </para>
-       </entry>
-       <entry>
-        <para>
-         Sel./file, dir.
-        </para>
-       </entry>
-       <entry>
-        <para>
-         Dir.
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         History
-        </para>
-       </entry>
-       <entry>
-        <para>
-         x
-        </para>
-       </entry>
-       <entry>
-        <para>
-         -
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         Hard Disk Space
-        </para>
-       </entry>
-       <entry>
-        <para>
-         --
-        </para>
-       </entry>
-       <entry>
-        <para>
-         o
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         GUI
-        </para>
-       </entry>
-       <entry>
-        <para>
-         o
-        </para>
-       </entry>
-       <entry>
-        <para>
-         -
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         Difficulty
-        </para>
-       </entry>
-       <entry>
-        <para>
-         o
-        </para>
-       </entry>
-       <entry>
-        <para>
-         +
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         Attacks
-        </para>
-       </entry>
-       <entry>
-        <para>
-         + (SSH)
-        </para>
-       </entry>
-       <entry>
-        <para>
-         +(SSH)
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         Data Loss
-        </para>
-       </entry>
-       <entry>
-        <para>
-         ++
-        </para>
-       </entry>
-       <entry>
-        <para>
-         +
-        </para>
-       </entry>
-      </row>
-     </tbody>
-    </tgroup>
-   </table>
-  </sect2>
- </sect1>
- <sect1 xml:id="sec.net.sync.cvs">
-  <title>Introduction to CVS</title><indexterm xml:id="idx.CVS" class="startofrange">
-
-  <primary>CVS</primary></indexterm><indexterm xml:id="idx.files_synchronizing_CVS" class="startofrange">
-
-  <primary>files</primary>
-
-  <secondary>synchronizing</secondary>
-
-  <tertiary>CVS</tertiary></indexterm>
-
-  <para>
-   CVS is suitable for synchronization purposes if individual files are edited
-   frequently and are stored in a file format, such as ASCII text or program
-   source text. The use of CVS for synchronizing data in other formats (such as
-   JPEG files) is possible, but leads to large amounts of data, because all
-   variants of a file are stored permanently on the CVS server. In such cases,
-   most of the capabilities of CVS cannot be used. The use of CVS for
-   synchronizing files is only possible if all workstations can access the same
-   server.
+   Rsync is useful when large amounts of data need to be transmitted regularly
```

Comment:
Rsync can be particularly useful when large amounts of data containing only minor changes need to be transmitted regularly. This is often the case when working with backups. Rsync can also be useful for mirroring staging servers that store complete directory trees of Web servers to a web server in a DMZ.